source/includes/_campaigns_and_cycles.md

```diff
@@ -1,6 +1,137 @@
 ## Campaigns APIs
 
+<internal>
+### **Campaigns Search**
```

Comment:
When I preview the document I saw ### **Campaigns Search**. To fix this problem it needs to add an empty row between `<internal>`(line 3) and  `### **Campaigns Search**`(line 4)